src/routing/decorator.tsx

```diff
@@ -1,5 +1,7 @@
-import { RouterContext } from 'next/dist/shared/lib/router-context'
-import Router, { NextRouter } from 'next/router'
+// this will be aliased by webpack at runtime (this is just for typing)
+import { RouterContext } from './resolved-router-context'
+// this will be aliased by webpack at runtime (this is just for typing)
+import Router from './resolved-router'
```

Comment:
@RyanClementsHax might be good to just alias whatever the latest nextjs path is and patch all previous versions

Reply:
That is definitely a way forward. I however don't have any previous versions I can use for that since I made the addon a week or so ago and the first version only included next v12 support. Do u see a way around this?